gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -199,9 +174,8 @@ public String selector() {
      * of fields that can be used.
      */
     public static RecordSetListOption fields(RecordSetField... fields) {
-      StringBuilder builder = new StringBuilder();
-      builder.append("nextPageToken,rrsets(").append(RecordSetField.selector(fields))
-          .append(')');
+      StringBuilder builder = selector(new StringBuilder().append("nextPageToken,rrsets("),
```

Comment:
Did you consider a special "list" selector method (which accepts in addition to the fields the "container" value and the "nextPageToken" unless the latter is common to all services)?